tests/test_openjdk.py

```diff
@@ -212,7 +212,7 @@ def test_jdk_cassandra(container_per_test):
 
     container_per_test.connection.run_expect(
         [0],
-        f"export JAVA_HOME=/usr/ && cd apache-cassandra-{cassandra_version}/ && bin/cassandra -R | tee {logs}",
+        f'export JAVA_HOME=/usr/ && export JVM_OPTS="\$JVM_OPTS -Xss2048k" && cd apache-cassandra-{cassandra_version}/ && bin/cassandra -R 2>&1 >{logs}',
```

Comment:
How large are the logs? If they are not overly huge, then I'd consider it much better style to perform this shell futzing in python directly, as you can get the stderr & stdeout from the `run_expect` call.